test/recipes/70-test_sslsigalgs.t

```diff
@@ -197,6 +199,29 @@ SKIP: {
     ok(TLSProxy::Message->success, "No TLSv1.2 sigalgs, ECDSA");
 }
 
+my ($dsa_status, $sha1_status, $sha224_status);
+SKIP: {
+    skip "TLSv1.3 disabled", 5 if disabled("tls1_3") || disabled("dsa");
```

Comment:
Hmm, probably a copy/paste-o, though I forget exactly how I wrote this line.
Fixed, and rebased for good measure.
Thanks for your patience on this one, @mattcaswell